include/Mixer.h

```diff
@@ -178,11 +178,11 @@ class LMMS_EXPORT FxMixer : public Model, public JournallingObject
 	// toChannel. NULL if there is no send.
 	FloatModel * channelSendModel(fx_ch_t fromChannel, fx_ch_t toChannel);
```

Comment:
`fx_ch_t` -> `channel_t` ?

Reply:
`channel` may mean different things in other context, so I will say no.

Reply:
So, `mix_ch_t`?